acapi/resources/acquiadata.py

```diff
@@ -62,8 +62,8 @@ def get_last_response(self):
         """Fetch the last response object. """
         return self.last_response
 
-    def request(self, uri=None, method='GET',
-                data=None, params={}, decode_json=True):
+    def request(self, uri=None, method='GET', headers={},
+                data=None, params={}, decode_json=True, stream=False):
```

Comment:
New arguments need to be appended to maintain backwards compatibility. Some code may be using positional arguments. We can't break that code.